client/src/App.js

```diff
@@ -12,22 +11,25 @@ import ProfileSettings from "./pages/ProfileSettings";
 
 import { AuthProvider } from "./contexts/user";
 import PrivateRoute from "./components/PrivateRoute/";
+import { ProfileProvider } from "./contexts/profile";
 
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
         <AuthProvider>
-          <Navbar />
-          <Route exact path="/" component={LandingPage} />
-          <Route exact path="/signup" component={SignUp} />
-          <Route path="/login" component={Login} />
-          <PrivateRoute path="/me" component={ProfileSettings} />
-          <PrivateRoute exact path="/become-a-sitter" component={LandingPage} />
-          <PrivateRoute exact path="/my-sitters" component={LandingPage} />
-          <PrivateRoute exact path="/my-jobs" component={LandingPage} />
-          <PrivateRoute exact path="/messages" component={LandingPage} />
+          <ProfileProvider>
+            <Navbar />
+            <Route exact path="/" component={ProfileSettings} />
+            <Route exact path="/signup" component={SignUp} />
+            <Route path="/login" component={Login} />
```

Comment:
The auth routes could be outside of the provider, right? Since the user isn't auth yet

Reply:
I think so, but @Braagaa can you confirm this would still work with your user integration?

Reply:
Just pushed a change to move those two outside but still in the AuthProvider

Reply:
Yep that's fine